submissions/7SergSmith7/HTML-CSS-Popup/styles.css

```diff
@@ -57,6 +64,7 @@ label:focus {
 .popup_btn-icon {
```

Comment:
Add styles to menu icon on `:focus` state.
![image](https://user-images.githubusercontent.com/16196199/96644269-78721600-1331-11eb-9dda-e7dec2ddecaf.png)
Also, add `cursor: pointer;` to this icon. It indicates to the user about an interactive element.

Reply:
Unfortunately this does not work (
Can't get it to work correctly